### DISCUSSION TITLE
Follow-up to adding new endpoints to InfluxDB

### DISCUSSION THREAD

**Discussion by PatRussell87** [Security-Onion-Solutions/securityonion#2688] [Unsupported Versions] [answered]
Follow-up question to [earlier post](https://github.com/Security-Onion-Solutions/securityonion/discussions/2589)

What is the best/proper method for copying the certs from the SO manager to the new telegraf endpoint? I know I can copy them but am not sure where to look and don't want to get the wrong ones.

**Comment by TOoSmOotH** [comment, marked as answer]
On the manager you need the following files:

/etc/pki/ca.crt
/etc/pki/influxdb.crt
/etc/pki/influxdb.key

If you look at the init file for telegraf you can see how they are mapped to the docker container.

https://github.com/Security-Onion-Solutions/securityonion/blob/master/salt/telegraf/init.sls#L63-L69

**Comment by PatRussell87** [comment]
Looks like there's and issue with the certs from the manager. Getting errors related to using legacy CN field and there are no SANs included in the cert. Are these errors correct or did I do something wrong (which is not out of the question)?
